package.json

```diff
@@ -8,14 +8,18 @@
     "lint": "eslint . --ext .vue,.js,.jsx,.cjs,.mjs --fix --ignore-path .gitignore"
   },
   "dependencies": {
+    "prettier": "^2.7.1",
```

Comment:
This one should be a dev dependency as we are only using prettier during the development to tidy up our code 😄

Reply:
Thanks, done! 😸